source/core/NumericString.cpp

```diff
@@ -498,6 +498,13 @@ void Format(SubString *format, Int32 count, StaticTypeAndData arguments[], Strin
                 ErrFormatMissingNumArgs(format, count, arguments, buffer, tempString);
             }
 
+            char replacementString[2*kTempCStringLength]; // temp used for modifying string to support opts for sigDigits, engineering notation, etc.
+            Int32 complexArg = 0; // when formatting complex data, 0==real, 1==imag, 2==finished
```

Comment:
Consider using an enumeration for those values, instead of having hardcode numbers.